.github/workflows/causal_lm_cpp.yml

```diff
@@ -737,71 +737,43 @@ jobs:
-          result = model.generate(**inputs, max_new_tokens=99)
-          decoded = processor.tokenizer.batch_decode(result[:, inputs["input_ids"].shape[1]:], skip_special_tokens=True)[0]
-          print(decoded)
-          with open("ref.txt", "w") as f:
-              f.write(f"question:\n{decoded}\n----------\nquestion:\n")
-
-      - name: Run visual_language_chat C++ sample - MiniCPM-V-2_6
+      - name: Run visual_language_chat C++ sample - tiny-random-minicpmv-2_6
         run: >
-          source ./ov/setupvars.sh
-          && ./build/samples/cpp/visual_language_chat/visual_language_chat ./MiniCPM-V-2_6/ ./images/
+          set -o pipefail
+          && source ./ov/setupvars.sh
+          && ./build/samples/cpp/visual_language_chat/visual_language_chat ./tiny-random-minicpmv-2_6/ ./images/
           <<< $'Describe the images?' | tee cpp.txt
         timeout-minutes: 2
-      - run: diff cpp.txt ref.txt
-      - name: Run visual_language_chat Python sample - MiniCPM-V-2_6
+      - name: Run visual_language_chat Python sample - tiny-random-minicpmv-2_6
         run: >
-          source ./ov/setupvars.sh
-          && ./samples/python/visual_language_chat/visual_language_chat.py ./MiniCPM-V-2_6/ ./images/
+          set -o pipefail
+          && source ./ov/setupvars.sh
+          && ./samples/python/visual_language_chat/visual_language_chat.py ./tiny-random-minicpmv-2_6/ ./images/
           <<< $'Describe the images?' | tee py.txt
         env:
           PYTHONPATH: "./build/"
-      - run: diff py.txt ref.txt
-      - name: Run visual_language_chat C++ sample with 2 prompts - MiniCPM-V-2_6
+      - run: diff cpp.txt py.txt
+      - name: Run visual_language_chat C++ sample with 2 prompts - tiny-random-minicpmv-2_6
         run: >
           source ./ov/setupvars.sh
-          && ./build/samples/cpp/visual_language_chat/visual_language_chat ./MiniCPM-V-2_6/ ./cat.jpg
-          <<< $'What is unusual on this image?\nGo on.' | tee cpp2.txt
+          && ./build/samples/cpp/visual_language_chat/visual_language_chat ./tiny-random-minicpmv-2_6/ ./cat.jpg
+          <<< $'What is unusual on this image?'
         timeout-minutes: 2
-      - name: Run visual_language_chat Python sample with 2 prompts - MiniCPM-V-2_6
+      - name: Run visual_language_chat Python sample with 2 prompts - tiny-random-minicpmv-2_6
         run: >
           source ./ov/setupvars.sh
-          && ./samples/python/visual_language_chat/visual_language_chat.py ./MiniCPM-V-2_6/ ./cat.jpg
-          <<< $'What is unusual on this image?\nGo on.' | tee py2.txt
+          && ./samples/python/visual_language_chat/visual_language_chat.py ./tiny-random-minicpmv-2_6/ ./cat.jpg
+          <<< $'What is unusual on this image?'
         env:
           PYTHONPATH: "./build/"
-      - run: diff cpp2.txt py2.txt
       - name: Download and convert LLaVa 1.5 model and an image
```

Comment:
can we split vision job into several ones? now, it's longest job on CI due to multiple models tested sequentially